index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <section>
+      <video id="buffVideo" controls>
+        <source
+          src="https://buffup-public.s3.eu-west-2.amazonaws.com/video/toronto+nba+cut+3.mp4"
```

Comment:
Use this Video file instead: 
https://buffup-public.s3.eu-west-2.amazonaws.com/video/Auto+Buff+Stevie+G.mp4

Reply:
Changed